storage/storage.go

```diff
@@ -1588,6 +1588,11 @@ type Query struct {
 	// which returns all properties. Passing ProjectionNoACL will omit Owner and ACL,
 	// which may improve performance when listing many objects.
 	Projection Projection
+
+	// If true, objects that end in exactly one instance of delimiter have
```

Comment:
I see this is copied from the API docs, but it should be rephrased based on how this is exposed via ObjectIterator in this library and based on godoc standards. I would say something like:

> IncludeTrailingDelimiter controls how objects which end in a single instance of Delimiter (for example, if Query.Delimiter = "/" and the object name is "foo/bar/") are included in the results. By default, these objects only show up as prefixes. If IncludeTrailingDelimiter is set to true, they will also be included as objects and their metadata will be populated in the returned ObjectAttrs.

Reply:
Done.